### PR TITLE
Removed bug button and made github external links

### LINKED ISSUE #621: Replace 'bug' button on member pages by a GitHub link at the bottom of the sidebar
The button is used once in a year, and links to a mailgroup.
A GitHub button would give members the possibility to create an issue themselves and would be less intrusive then a button that blocks a part of the page

### PR DESCRIPTION
**Pull request by JustNireon**
resolves #621 

Removed the bug button and added an external link to svsticky GitHub so if people want to report issues they can do it there

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -684,6 +684,7 @@ en:
       declarations: DigiDecs
       books: Books
       vacancies: Vacancies
+      github: Github
```
Comment (Mstiekema) [inline config/locales/en.yml:687]: It's `GitHub`

File: config/locales/nl.yml [hunk 2]
```diff
@@ -628,6 +628,7 @@ nl:
       declarations: DigiDecs
       books: Boeken
       vacancies: Vacatures
+      github: Github
```
Comment (Mstiekema) [inline config/locales/nl.yml:631]: It's `GitHub`

File: app/views/layouts/members.html.haml [hunk 3]
```diff
@@ -74,7 +74,9 @@
               = link_to "https://svsticky.nl/carriere/vacatures", :class => 'list-group-item list-group-item-action' do
                 %span= I18n.t("members.navigation.vacancies")
                 %i.fa.fa-briefcase
-            -# Signout button for mobile
```
Comment (Riscky) [inline app/views/layouts/members.html.haml:77]: why did you remove this comment?
